scalajvm/app/controllers/Licenses.scala

```diff
@@ -0,0 +1,10 @@
+package controllers
+
+import helpers.ActionWithTx
+import play.api.mvc._
+
+object Licenses extends Controller {
+  def listLicenses() = ActionWithTx {
```

Comment:
We don't need a transaction to just render a static page. Please use `Action` here instead of `ActionWithTx`.